src/connectors/configDb/configDb.connector.js

```diff
@@ -69,6 +99,18 @@ const getConfigVersion = async regDataVersion => {
   }
 };
 
+const findCouncilById = async (collection, id) => {
```

Comment:
these make sense - seems like we could use them in registration.service insead of returning all